src/NexusMods.App.UI/Dialog/ICloseable.cs

```diff
@@ -0,0 +1,8 @@
+using System.Reactive;
+using ReactiveUI;
+
+namespace NexusMods.App.UI.Dialog;
+
+public interface ICloseable
```

Comment:
Closable sounds better than Closeable
```suggestion
public interface IClosable
```